spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/storage/GcpStorageAutoConfiguration.java

```diff
@@ -48,6 +49,7 @@
  */
 @Configuration
 @ConditionalOnClass({ GoogleStorageProtocolResolverSettings.class, Storage.class })
+@ConditionalOnProperty(value = "spring.cloud.gcp.storage.enabled", matchIfMissing = true)
```

Comment:
If we add some non-managed configuration property, it has to be included into the `additional-spring-configuration-metadata.json`: https://github.com/spring-cloud/spring-cloud-gcp/issues/973